scripts/eslint/rules/no-in-operator.cjs

```diff
@@ -1,5 +1,8 @@
 const { createRule } = require("./utils.cjs");
 
+/** @import { TSESTree } from "@typescript-eslint/utils" */
+void 0;

```

Comment:
It's super annoying that we need this.

Reply:
Yep; I could leave it floating but then it'd be attached to some other random declaration.